components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/JITProvisioningPostAuthenticationHandler.java

```diff
@@ -334,6 +338,15 @@ private PostAuthnHandlerFlowStatus handleRequestFlow(HttpServletRequest request,
                         username = sequenceConfig.getAuthenticatedUser().getUserName();
                         isUserCreated = true;
                     }
+                    // Check if the associated local account is locked.
```

Comment:
how do we lock the user if the user is trying continuos invalid codes?

Reply:
It is handled by the fix: https://github.com/wso2-extensions/identity-outbound-auth-totp/pull/73/files